docs/pages/preview/dev-process-overview.mdx

```diff
@@ -0,0 +1,170 @@
+
+However, both Expo Go and Snack are not intended for building production apps. They are great when you're just getting started on a project or for prototypes, but if you plan on deploying your app to the store, then development builds will provide a more flexible, reliable, and complete development environment. **This guide does not go into any detail about Expo Go, and this is the only section that mentions it.**
+
+</Collapsible>
+
+<Collapsible summary="Development builds">
+
+TODO
+
+</Collapsible>
+
+<Collapsible summary={`Android and iOS native projects`}>
+
+React Native apps for mobile platforms are made of two interconnected parts:
+
+
+<Collapsible summary={`1. The app JavaScript`}>
+
+This contains your React components and most, if not all, of your application logic. It has more or less the same role as the app JavaScript on React websites.
+
+</Collapsible>
+
+<Collapsible summary={`2. The native projects`}>
+
+Android and Xcode projects that bundle the JavaScript app, serve as the launchpad for the JavaScript app on each platform, handle the rendering of native components, and provide the means to access platform-specific functionalities and integrate with any installed native libraries. App configuration, such as the name (as it appears on the home screen), icon, required permissions, associated domains, supported orientations, and so on, is configured in the native project.
+
+</Collapsible>
+
+Like any mobile app, the application that is distributed to users is created by compiling (”building”) the Xcode or Android project.
+
+When you initialize a new app with `npx create-expo-app`, you will not see any **android** or **ios** directories. You can generate the native projects by running `npx expo prebuild`, which will initialize the native projects and then apply the project Expo app config (**app.json/app.config.js**) to them ([learn more](https://docs.expo.dev/workflow/prebuild/)).
+
+If you use a cloud-based development workflow, you may never need to run prebuild or install Xcode or Android Studio on your own machine (although you may find this to be useful). This is explained below in the “Local and cloud-based development workflows” section.
+
+<Collapsible summary={`Why aren’t the native projects created by default when you initialize a project with create-expo-app?`}>
+
+The default behavior encourages the use of [Continuous Native Generation](/workflow/continuous-native-generation/) (CNG) to generate the native projects when needed, which can make upgrades and project maintenance significantly easier.
+The following three commands result in more or less the same project:
+
+- `npx create-expo-app MyApp && cd MyApp && npx expo prebuild`
+- `npx create-expo-app --template bare-minimum`
+- `npx react-native init MyApp && cd MyApp && npx install-expo-modules`
+
+</Collapsible>
+
+</Collapsible>
+
+<Collapsible summary={`Continuous Native Generation (CNG)`}>
```

Comment:
Would it be too long to somehow add "should I commit the `ios` & `android` directories" in the title?